MovieApp/MovieApp/App/Features/Movies/ViewControllers/SearchMoviesViewController/MoviesSearchViewController.swift

```diff
@@ -24,24 +24,30 @@ class MoviesSearchViewController: UIViewController {
         
         buildViews()
         styleNavigationController()
-        presenter.setMoviesViewDelegate(moviesViewDelegate: self)
-        presenter.fetchMovies()
+        presenter.setDelegate(delegate: self)
         searchBarStackView.setDelegate(delegate: self)
+        searchBarStackView.searchBar.searchTextField.delegate = self
+        searchBarStackView.searchBar.searchTextField.addTarget(self, action: #selector(queryMovies), for: .editingChanged)
         searchBarStackView.cancelButton.addTarget(self, action: #selector(popViewController), for: .touchUpInside)
```

Comment:
make a separate func out of this. This are 4 lines of code connected with searchBarStackView